utils/create-component/createReactComponent.js

```diff
@@ -22,12 +27,11 @@ const cwd = process.cwd();
 module.exports = (modulePath, name, description, prerelease, category) => {
   const moduleName = `@workday/canvas-kit-${prerelease && prerelease + '-'}react/${name}`;
 
-  console.log('\nCreating ' + chalk.blue.underline(`${moduleName}\n`));
+  consoleMessage('\nCreating', `${moduleName}\n`);
```

Comment:
Q: Is changing blue underline to cyan intentional?

Reply:
I just thought it looked nicer. The blue was kinda harsh :D 
<img width="417" alt="Screen Shot 2022-08-10 at 5 50 22 PM" src="https://user-images.githubusercontent.com/1562615/184047013-6e8a2bd1-3896-45ba-bac1-bcbe63556e3b.png">

Reply:
<img width="528" alt="Screen Shot 2022-08-10 at 5 51 37 PM" src="https://user-images.githubusercontent.com/1562615/184047120-61e3e2b8-bc0d-469f-b819-5a765cb9c515.png">